corgi/tasks/management/commands/loadproductdata.py

```diff
@@ -1,56 +1,81 @@
+from django.conf import settings
 from django.core.management.base import BaseCommand
 
 from corgi.tasks.brew import load_brew_tags
 from corgi.tasks.errata_tool import load_et_products
 from corgi.tasks.prod_defs import update_products
 from corgi.tasks.pulp import setup_pulp_relations, update_cdn_repo_channels
 from corgi.tasks.rhel_compose import save_composes
+from corgi.tasks.yum import load_yum_repositories
 
 
 class Command(BaseCommand):
 
     help = "Fetch product data from Product Definitions."
 
     def handle(self, *args, **options):
+        if settings.COMMUNITY_MODE_ENABLED:
+            self.do_update_products()
+            self.do_load_yum_repositories()
+
+            # TODO load_brew_tags, can't find
+        else:  # Enterprise mode
+            self.do_load_et_products()
+            self.do_update_products()
+            self.do_update_cdn_repo_channels()
+            self.do_save_composes()
+            self.do_load_brew_tags()
+            self.do_load_yum_repositories()
+            self.delay_setup_pulp_relations()
```

Comment:
I think in enterprise mode, load_yum_repostories should be called with .delay and run after setup_pulp_relations. The Yum repo tasks / collector works better when Channels are already set up, so should always run after the Pulp repo tasks / collector:
```suggestion
            self.delay_setup_pulp_relations()
            self.delay_load_yum_repositories()
```

In community mode we obviously won't have real channels...I don't know what the rest of our data will actually look like in that case, but I think the Yum repo data should be OK. The collector will create a fake Channel based on the Yum repo's URL, if a Channel with a matching URL doesn't already exist.

Reply:
Ok, I'll make that change.